backend/migrations/versions/9f6a80078e7f_add_userhint_table_with_relationships.py

```
@@ -0,0 +1,55 @@
+down_revision = 'd65d4263d860'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('user_hints',
+    sa.Column('user_id', sa.Integer(), nullable=False),
+    sa.Column('hint_id', sa.Integer(), nullable=False),
+    sa.Column('unlocked', sa.Boolean(), nullable=True),
+    sa.ForeignKeyConstraint(['hint_id'], ['hints.id'], ),
```

Ruff (0.8.2)

Trailing comma prohibited

Remove trailing comma

(COM819)